### DISCUSSION TITLE
HA Phone BLE beacon no longer detected?

### DISCUSSION THREAD

**Discussion by fokcuk** [custom-components/ble_monitor#959] [General]
Hello
I have noticed that my mobile's HA app BLE beacon is no longer detected by BLE Monitor.
How do I troubleshoot it or request for the feature to be added?

**Comment by Ernst79** [comment]
How did you add it to BLE monitor? Did you configure the UUID address as a device and did you enable "track device" in the BLE monitor options?

**Reply by fokcuk** [reply]
It found it by itself long time ago. I was not using it till today. Went back and saw that it had not updated for a long time

Can you please help me add it back? I removed the entity hoping it would come back but it did not

**Comment by Ernst79** [comment]
It should normally be added automatically, if you have enabled "discovery" ("Discover devices and sensors automatically" in the UI) in the BLE monitor settings. But as it didn't update your entities, it probably won't be found as well. It will normally be added as soon as the first message arrives. But check this "discovery" setting to be enabled first

**Comment by fokcuk** [comment]
I found it in the HA log. Said that it was disabled, so was not tracking. Re-enabled it, but now have this:

`DEBUG (MainThread) [homeassistant.components.device_tracker] Entity device_tracker.ble_tracker_55b561a9e94c40a9925f992f69408524 unexpectedly has a device info`

**Reply by Ernst79** [reply]
hmmm, never seen that message. Will check it later on my end
